### PR TITLE
Add display documentation

### PR DESCRIPTION
**Pull request by v-mingho**
Task: task-[421681](https://dev.azure.com/ceapex/Engineering/_boards/board/t/Fox%20Team/Stories/?workitem=421681)

Link: preview-[187](https://design.docs.microsoft.com/pulls/187/atomics/display.html)

This PR adds documentation for display atomics.

## Testing

1. Review the link above.

### CODE REVIEW

File: site/src/atomics/display.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, and `none`.
```
Comment (wibjorn) [inline site/src/atomics/display.md:9]: Doesn't quite match: `$displays: 'block', 'flex', 'inline', 'inline-block', 'inline-flex', 'grid', 'none' !default;`

File: site/src/atomics/display.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+The following breakpoints are supported for each display value:
+
+```Text
+.position-{value}-tablet
+.position-{value}-desktop
+.position-{value}-widescreen
+```
+
+## Block
```
Comment (wibjorn) [inline site/src/atomics/display.md:19]: Do we need six different code examples to explain one pattern?

File: site/src/atomics/display.md (hunk 1 above)
Comment (v-mingho) [reply site/src/atomics/display.md]: Updated

File: site/src/atomics/display.md (hunk 2 above)
Comment (v-mingho) [reply site/src/atomics/display.md]: Reduced to one.

File: site/src/atomics/display.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+.display-{value}-desktop
+.display-{value}-widescreen
+```
+
+## Example
+
+```html
+<div class="border">
+	<button class="button display-block border">Foo</button>
+	<button class="button display-block border">Bar</button>
```
Comment (wibjorn) [inline site/src/atomics/display.md:24]: \<button class="display-block">Foo<\/button>
\<button class="display-block-tablet">Foo<\/button>

?

Comment (v-mingho) [reply site/src/atomics/display.md]: What about something like this?

```
<div class="display-block border">
	<div>Foo</div>
	<div>Foo</div>
</div>
<div class="display-flex-tablet border">
	<div>Bar</div>
	<div>Bar</div>
</div>
<div class="display-inline-desktop border">
	<div>Baz</div>
	<div>Baz</div>
</div>
<div class="display-inline-flex-widescreen border">
	<div>Foo</div>
	<div>Foo</div>
</div>
```

File: site/src/atomics/display.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, and `none`.
+
+The following breakpoints are supported for each display value:
```
Comment (wibjorn) [inline site/src/atomics/display.md:11]: Re-arrange? On the current spacing documentation page (in PR) this example would be the last.

File: site/src/atomics/display.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+```Text
+.display-{value}-tablet
+.display-{value}-desktop
+.display-{value}-widescreen
+```
+
+## Example
```
Comment (wibjorn) [inline site/src/atomics/display.md:19]: Can we make this more uniform with the other article? Is this `Usage`?

File: site/src/atomics/display.md (hunk 4 above)
Comment (v-mingho) [reply site/src/atomics/display.md]: Updated to match the new format in the spacing doc PR.

File: site/src/atomics/display.md (hunk 5 above)
Comment (v-mingho) [reply site/src/atomics/display.md]: Updated

File: site/src/atomics/display.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+Display atomics are used to set the display behavior of an element. 
+## Class Pattern
```
Comment (wibjorn) [inline site/src/atomics/display.md:10]: nit: space

Comment (v-mingho) [reply site/src/atomics/display.md:11]: Space added

**Review by wibjorn — DISMISSED** [review]

File: site/src/atomics/display.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+</div>
+<div class="display-inline-desktop border">
+	<div>Baz</div>
+	<div>Baz</div>
+</div>
+<div class="display-inline-flex-widescreen border">
+	<div>Foo</div>
+	<div>Foo</div>
+</div>
+```
```
Comment (wibjorn) [inline site/src/atomics/display.md:44]: The above looks great, should we wait until the code block changes hit and add an `Available classes` section at the end of this?

Comment (v-mingho) [reply site/src/atomics/display.md:44]: Yes, I'll need to revisit after the code block changes are checked in.

Comment (ollips) [reply site/src/atomics/display.md:44]: What do you think about splitting one large usage example into 4 smaller ones?

In this particular case, it feels hard for me to read the examples: different position classes are assigned to blocks, as well as different screen sizes, and it all make examples to be too close to each other and hard to understand what class is assigned to what section 😵

image

Comment (v-mingho) [reply site/src/atomics/display.md:53]: I think that makes sense, updated.

**Review by wibjorn — DISMISSED** [review]

**Comment by v-mingho** [comment]
Waiting for the outcome of https://github.com/microsoft/atlas-design/pull/207 , then will update if necessary.

File: site/src/atomics/display.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+	<div>Bar</div>
+	<div>Bar</div>
+</div>
+```
+
+```html
+<div class="display-inline-desktop border">
+	<div>Baz</div>
+	<div>Baz</div>
+</div>
```
Comment (ollips) [inline site/src/atomics/display.md:41]: shouldn't the `display-inline-desktop` class be applied to the inner `div`'s?

Comment (v-mingho) [reply site/src/atomics/display.md:41]: Ah yes, updated

**Review by ollips — APPROVED** [review]

**Review by wibjorn — APPROVED** [review]

**Pull request merged**